examples/all_the_things.py

```diff
@@ -89,25 +90,28 @@ def dimensions(test):
 
 
 def attachments(test):
-  test.Attach('test_attachment', 'This is test attachment data.')
-  test.AttachFromFile('example_attachment.txt')
+  test.attach('test_attachment', 'This is test attachment data.')
```

Comment:
This is just the beginning; the pylint and restructuring PR's are going to require every test script to get a pretty thorough once-over. Sorry! :-\ I believe it's better to do this in one fell swoop now (just prior to publishing in PyPI) and have the API be fairly stable once we go live with v1.0 on PyPI.